bin/redis-commander.js

```diff
@@ -414,6 +481,67 @@ function createConnectionObjectFromArgs(argList) {
     if (argList['redis-tls']) {
       connObj.tls = {};
     }
+    if (argList['redis-tls'] &&
+      (argList['redis-tls-ca-cert-file'] || argList['redis-tls-ca-cert']
+        || argList['redis-tls-cert-file'] || argList['redis-tls-cert']
+        || argList['redis-tls-key-file'] || argList['redis-tls-key']
+        || argList['redis-tls-server-name'])) {
+      connObj.tls = {};
```

Comment:
As the first part of the IF is the same as line 481 above this can be moved into the IF clause there and no extra IF clause with &&.
Therefore only "redis-tls" is set - than `tls` is an empty object as before or one of your new parameters is set and than `tls` gets more complex as needed.